src/rust/ide/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -184,7 +189,28 @@ impl View {
         scene.views.main.remove(&shape_system.shape_system.symbol);
         scene.views.viz.add(&shape_system.shape_system.symbol);
 
-        Self {logger,display_object,background,overlay}
+        let shape_system = scene.shapes.shape_system(PhantomData::<overlay::Shape>);
+        scene.views.main.remove(&shape_system.shape_system.symbol);
+        scene.views.viz.add(&shape_system.shape_system.symbol);
+
+        let div            = web::create_div();
+        let background_dom = DomSymbol::new(&div);
+        // TODO : We added a HTML background to the `View`, because native background was overlapping
+        //        the JS visualization. This should be further investigated while fixing rust
+        //        visualization displaying. (#796)
+        background_dom.dom().set_style_or_warn("width"         ,"0"        ,&logger);
+        background_dom.dom().set_style_or_warn("height"        ,"0"        ,&logger);
+        background_dom.dom().set_style_or_warn("z-index"       ,"1"        ,&logger);
+        background_dom.dom().set_style_or_warn("overflow-y"    ,"auto"     ,&logger);
+        background_dom.dom().set_style_or_warn("overflow-x"    ,"auto"     ,&logger);
+        background_dom.dom().set_style_or_warn("background"    ,"#312F30FA",&logger);
+        background_dom.dom().set_style_or_warn("border-radius" ,"14px"     ,&logger);
+        background_dom.dom().set_style_or_warn("pointer-events","auto"     ,&logger);
```

Comment:
There is no point in setting "pointer-events", as we won't handle any events in background anyway.